cron/internal/config/config_test.go

```diff
@@ -37,8 +37,9 @@ const (
 	prodShardSize           int    = 10
 	prodMetricExporter      string = "stackdriver"
 	// Raw results.
-	prodRawBucket        = "gs://ossf-scorecard-rawdata"
-	prodRawBigQueryTable = "scorecard-rawdata"
+	prodRawBucket                = "gs://ossf-scorecard-rawdata"
+	prodRawBigQueryTable         = "scorecard-rawdata"
+	prodBigQueryExportsBucketURL = "gs://ossf-scorecard-bq-export-results"
```

Comment:
Badges data gets stored in `gs://ossf-scorecard-results`. For the sake of consistency could you name the bucket `gs://ossf-scorecard-cron-results`?

Reply:
I have fixed it. Thanks